pyaedt/edb_core/EDB_Data.py

```diff
@@ -2663,9 +2663,26 @@ def refdes(self):
         """
         return self.edbcomponent.GetName()
 
+    @property
+    def is_enabled(self):
+        """Check if the current object is enabled.
+
+        Returns
+        -------
+        bool
+            ``True`` if current object is enabled, ``False`` otherwise.
+        """
+        return self.edbcomponent.GetComponentProperty().IsEnabled()
+
+    @is_enabled.setter
+    def is_enabled(self, enabled):
+        """Enables the current object."""
+        if self.edbcomponent.GetComponentType() in [1, 2, 3]:
```

Comment:
you need to use the right enumerator now not Integer, e.g Edb.Definition.ComponentType.Resistor